scripts/choco-install.ps1

```diff
@@ -0,0 +1,38 @@
+$nodejs = "nodejs.install"
+$python = "python3"
+
+# Check for existence of required environment variables
+if ( $env:ChocolateyInstall -eq $null ) {
+  [Console]::Error.WriteLine('Missing $env:ChocolateyInstall environment variable')
+  exit 1
+}
+
+# Add the cached packages with source priority 1 (Chocolatey community is 0)
+New-Item -Path "${PSScriptRoot}\..\tmp\chocolatey" -ItemType "directory" -ErrorAction:SilentlyContinue
+choco source add --name="cache" --source="${PSScriptRoot}\..\tmp\chocolatey" --priority=1
+
+# Install nodejs v16.14.2 (will use cache if exists)
+choco install "$nodejs" --version="16.14.2" --require-checksums -y
```

Comment:
Try and see what `--checksum` it does.

Reply:
I've had a go at using `--checksum` with a garbage checksum and it doesn't seem to prevent the package from being downloaded. I'm not sure if it would cause an error during installation - I would need to try this on the CI/CD since I don't seem to have the permissions to uninstall/install packages on `matrix-win-1`.

Reply:
Since checking package downloads against a checksum isn't supported yet (https://github.com/chocolatey/choco/issues/1144) I'll leave it as `--require-checksums`. This at least ensures that installed packages don't download any additional requirements that aren't protected by a checksum.